lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -93,20 +93,22 @@ export async function checkOnboardingBranch(
         { branch: onboardingBranch, commit, onboarding: true },
         'Branch created'
       );
-
-      // set onboarding branch cache
-      setOnboardingCache(
-        config.onboardingBranch!,
-        getBranchCommit(config.defaultBranch!)!,
-        commit
-      );
     }
   }
   if (!GlobalConfig.get('dryRun')) {
-    logger.debug('Checkout onboarding branch.');
     // TODO #7154
-    await scm.checkoutBranch(onboardingBranch!);
+    if (!isConflicted) {
+      logger.debug('Merge onboarding branch in default branch');
+      await mergeBranch(onboardingBranch!, true);
```

Comment:
should this go through scm interface for gerrit support? 🤔

Reply:
Not sure, but if it does then its a task for another PR.

Reply:
ok